### DISCUSSION TITLE
New Device "Iluminize 5120.2110"

### DISCUSSION THREAD

**Discussion by widder17** [Koenkk/zigbee2mqtt#16849] [Show and tell]
Hello,
I configured the dimmer with switching output (5120.2110) from Iluminize.
The range of functions is the same as with the switch dimmer from LED_Trading (HK-LN-DIM-A).
All functions can be switched easily under ioBroker Zigbee.

Unfortunately I don't know how to upload the configuration to Github, so I'm making it available to you here in the hope that someone will add it to Github.

Under ioBroker I added the following to the "/opt/iobroker/node_modules/iobroker.zigbee/lib" folder and then to the "devices.js" files:

 **_{
        models: [' 5120.2110'],
        icon: 'img/illuminate_511_201.png',
 },_**

In the folder "opt/iobroker/node_modules/zigbee-herdsman-converters/devices" I have in the file "iluminize.js" added the following:

 _**},
    {
        zigbeeModel: ['5120.2110'],
        model: '5120.2110',
        vendor: 'Iluminize',
        description: 'ZigBee 3.0 dimming actuator mini 1x 230V',
        extend: extend.light_onoff_brightness({noConfigure: true}),
        configure: async (device, coordinatorEndpoint, logger) => {
            await extend.light_onoff_brightness().configure(device, coordinatorEndpoint, logger);
            const endpoint = device.getEndpoint(1);
            await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
            await reporting.onOff(endpoint);
        },**_

I hope that it is possible to include the adapter with the information.

Best regards

Jörg

**Comment by Koenkk** [comment]
Added!
